src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -93,9 +160,10 @@ public String getAllianceColor() {
   }*/
 
   public String senseColor() {
-    Color color = colorSensorOne.getColor();
-    double redAmount = color.red;
-    double blueAmount = color.blue;
+
+    Color[] colors = colorSensors.getColors();
+    double redAmount = colors[0].red;
+    double blueAmount = colors[0].blue;
```

Comment:
Can we throw a comment here why 0 is the only address we're reading, in case these get moved in the future?

Reply:
0 means that we're only sensing color for the sensor under the centering wheels (the sensor leading into the CDS). We could potentially expand this for multiple sensors, but for now, it makes the most sense to only sense color for the first sensor. I'll be sure to add an in-code comment addressing this.